packages/apidom-parser/src/types.d.ts

```diff
@@ -19,7 +19,7 @@ export interface ApiDOMParserAdapter {
 
 export interface ApiDOMParser {
```

Comment:
No need for this interface anymore.

When you define
```ts
class ApiDOMParser {
...
}
```
then this code automatically generates instance type and class type in `type scope`.

**Proof**

```ts
type InstanceType = ApiDOMParser;
type ClassType = typeof ApiDOMParser;
```